composer.json

```diff
@@ -3,12 +3,17 @@
     "license": "OPL-1.0",
     "require": {
         "php": "^8.1",
+        "jangregor/phpstan-prophecy": "^1.0",
         "phpdocumentor/guides-cli": "dev-main",
         "phpdocumentor/guides-theme-bootstrap": "dev-main",
+        "phpstan/extension-installer": "^1.3",
         "symfony/http-client": "^6.3"
     },
     "require-dev": {
         "ergebnis/composer-normalize": "^2.30",
+        "phpstan/phpstan": "^1.10",
+        "phpstan/phpstan-strict-rules": "^1.5",
+        "phpstan/phpstan-webmozart-assert": "^1.2",
```

Comment:
Why this one, do we use webmozart/assert?

Reply:
removed it until we need it